providers/src/airflow/providers/amazon/aws/auth_manager/aws_auth_manager.py

```diff
@@ -76,12 +76,12 @@ class AwsAuthManager(BaseAuthManager):
 
     Leverages AWS services such as Amazon Identity Center and Amazon Verified Permissions to perform
     authentication and authorization in Airflow.
-
-    :param appbuilder: the flask app builder
     """
 
-    def __init__(self, appbuilder: AirflowAppBuilder) -> None:
-        super().__init__(appbuilder)
+    appbuilder: AirflowAppBuilder | None = None
+
+    def __init__(self) -> None:
+        super().__init__()
```

Comment:
```suggestion
    def __init__(self, *args, **kwargs) -> None:
        super().__init__(*args, **kwargs)
```

should make it compatible?

Reply:
It wont be compatible anyway because of the many breaking changes in the auth manager interface so I would not bother caring about the compatibility

Reply:
Fair enough. Just silently breaking this for (potential) AF2 users isn't right either though. Can we fix compatibility for the other stuff we've broken already?

Reply:
Yep, I'll update this PR to make it only compatible with Airflow 3 (compat tests are failing because of that anyway)

Reply:
> Can we fix compatibility for the other stuff we've broken already?

We have not broken anything yet, this is the first PR

Reply:
And the AWS auth manager is still marked as experimental, so it wont be technically a breaking change for the Amazon provider package